test/unit/jasmine-boot.js

```diff
@@ -197,26 +199,26 @@ function initializePDFJS(callback) {
   // Sets longer timeout.
   jasmine.DEFAULT_TIMEOUT_INTERVAL = 30000;
 
-  // Replace the browser window's `onload`, ensure it's called, and then run
-  // all of the loaded specs. This includes initializing the `HtmlReporter`
-  // instance and then executing the loaded Jasmine environment.
-  var currentWindowOnload = window.onload;
-
-  window.onload = function () {
-    if (currentWindowOnload) {
-      currentWindowOnload();
+  function extend(destination, source) {
+    for (var property in source) {
```

Comment:
`var` -> `const` now that my PR just landed (which created a merge conflict that needs to be addressed; sorry for that!)

Reply:
No worries!